db/migrate/20150715184131_create_attribute_mappers.rb

```
@@ -0,0 +1,10 @@
+class CreateAttributeMappers < ActiveRecord::Migration
+  def change
+    create_table :attribute_mappers do |t|
+      t.integer :mapping_direction, default: 0, null: false
```

I figured from reading the enum docs that defaulting would be a good idea. This will be treated as an `:import`. I take it you'd prefer to not set a default and if it's not set, let it fail?